src/lambda/graphql.js

```diff
@@ -5,5 +5,9 @@ exports.handler = async (request, context) => {
   await setup.run();
   const response = await handleRequest(request, context);
 
+  response.headers = {
+    'Access-Control-Allow-Origin': '*',
+    'Access-Control-Allow-Headers': 'Origin, X-Requested-With, Content-Type, Accept'
+  }
```

Comment:
Apollo Server actually provides an easy way to set the CORS headers and set the `access-control-allow-origin` value to fix this local CORS issue. We don't need to do it explicitly here. 

In our `src/lambda/lib/apollo.js` file we can simply update the constructor to pass in the CORS object with the following settings:

<img width="553" alt="image" src="https://user-images.githubusercontent.com/1451729/66575982-28644f00-eb34-11e9-9c63-a0af0c7abd5d.png">

Feel free to make the updates and of course update your Apollo client to point to your local host url where the GraphQL endpoint exists, which should be `http://localhost:8888/.netlify/functions/graphql`

Lastly, as a reminder, you want to be running the project using Netlify Dev, by using the following command `ntl dev`. This will run *both* your react front-end your backend. Let me know if you have any questions!

Reply:
Change applied, ready for another review.